recipes/agent_windows.rb

```diff
@@ -1,3 +1,32 @@
 # Installs and configures the New Relic Infrastructure agent on Windows
+#
+# Chef uses sha256 checksums, to generate this, I downloaded the file
+# and on OS X did 'shasum -a 256 newrelic-infra.msi' YMMV on other
+# platforms
+#
+# TODO: pin to specific version
+# TODO: abstract checksum
+# TODO: make install location configurable
 
-raise 'Windows chef support for New Relic Infrastructure agent not yet available'
+windows_package 'newrelic-infra' do
+  action :install
+  source 'https://download.newrelic.com/infrastructure_agent/windows/newrelic-infra.msi'
+  checksum '3c9f98325dc484ee8735f01b913803eaef54f06641348b3dd9f3c0b3cd803ace'
```

Comment:
Since the source is pulling down the "latest" msi, the checksum is likely different on every run when the agent installer has been updated.

Reply:
@chaupt `windows_package` is part of core Chef, and will be supported for quite a long time. I think I've figured out a good `default` for people that will always grab the latest, skip the checksum, and also allow people to pin to a particular version and do checksum validations as well. I'll push that here shortly.